src/routes/api.php

```diff
@@ -29,6 +29,6 @@
 //     return response()->json(['Laravel 7 CORS Demo']);
 // });
 
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
-//     return $request->user();
-// });
+Route::middleware('auth:api')->get('/user', function (Request $request) {
+    return $request->user();
+});
```

Comment:
1-Make `UserController.php` for future work
2-Add this function to user controller and link this route to controller function 
3-Remove commented code in this file